_projects/SP19-CITL_VR_Lecture

```diff
@@ -0,0 +1,14 @@
+---
+layout: project
+title: CITL VR Lecture
+image: <link to project image (optional)>
+desc: |
+ <description>
+github: <github link (optional)>
+dl: https://drive.google.com/drive/folders/1VLgqbK16jDJLeW2mQswaC2PS33gWzUJG?usp=sharing
+semester: Spring 2019
+team: 2
+youtube: The video was uploaded to Piazza
```

Comment:
This is parsed as a link, either post it on YouTube and copy the video ID or remove this line
```suggestion
```